### PR TITLE
cloudfront distributions regenerated and tags added

### PR DESCRIPTION
**Pull request by amanenk**

### CODE REVIEW

**Review by bbernays — APPROVED** [review]
Overall it looks good, but are we not grabbing any data from `GetDistribution` as not all information is available from `ListDistributions`

See this issue: #305

File: resources/cloudfront_distributions.go [hunk 1]
```diff
@@ -488,73 +710,126 @@ func fetchCloudfrontDistributions(ctx context.Context, meta schema.ClientMeta, _
...
+}
+func resolveCloudfrontDistributionsAliasIcpRecordals(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	distribution, ok := resource.Item.(types.DistributionSummary)
+	if !ok {
+		return fmt.Errorf("not types.DistributionSummary")
+	}
+	j := map[string]interface{}{}
+	for _, a := range distribution.AliasICPRecordals {
+		j[*a.CNAME] = a.ICPRecordalStatus
+	}
```
[the remark this answers is not in the document]
Comment (amanenk) [reply resources/cloudfront_distributions.go]: Could you explain more?

**Comment by amanenk** [comment]
I have regenerated with GetDistribution result

File: resources/cloudfront_distributions.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (bbernays) [reply resources/cloudfront_distributions.go]: From the docs this field seems to be solely about chinese regulations that might not be relevant to other regions

`AWS services in China customers must file for an Internet Content Provider (ICP) recordal if they want to serve content publicly on an alternate domain name, also known as a CNAME, that they've added to CloudFront`

**Review by bbernays — APPROVED** [review]
LGTM

File: resources/cloudfront_distributions.go [hunk 2]
```diff
@@ -14,447 +14,712 @@ import (
...
-				Resolver: schema.PathResolver("IsIPV6Enabled"),
+				Name:        "http_version",
+				Description: "(Optional) Specify the maximum HTTP version that you want viewers to use to communicate with CloudFront",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("DistributionConfig.HttpVersion"),
 			},
 			{
-				Name: "last_modified_time",
-				Type: schema.TypeTimestamp,
+				Name:        "ip_v6_enabled",
```
Comment (roneli) [inline resources/cloudfront_distributions.go:210]: ip_v6 -> ipv6

**Review by roneli — CHANGES_REQUESTED** [review]
Required to drop tables that turned into jsons.

**Review by spangenberg — APPROVED** [review]
LGTM

**Review by roneli — APPROVED** [review]

**Pull request merged**